packages/target-decisioning-engine/src/ruleEvaluator.js

```diff
@@ -3,16 +3,21 @@ import jsonLogic from "json-logic-js";
 import { isDefined } from "@adobe/target-tools";
 import { createMboxContext, createPageContext } from "./contextProvider";
 import { computeAllocation, getOrCreateVisitorId } from "./allocationProvider";
-import { cloneDeep } from "./utils";
+import { cloneDeep, parseDomainBasic } from "./utils";
 import { ACTIVITY_ID } from "./constants";
 
 /**
  *
  * @param { String } clientId
  * @param { import("@adobe/target-tools/delivery-api-client/models/VisitorId").VisitorId } visitorId
+ * @param {Function} parseDomainImpl
```

Comment:
the type for `parseDomainImpl` should be `import("../types/DecisioningContext").ParseDomainFunc` instead of `Function`.